samples/main.cpp

```diff
@@ -42,22 +42,29 @@ std::unique_ptr<NovelRT::Input::BasicInteractionRect> memeInteractionRect;
 
 int main(int argc, char *argv[])
 {
+  std::filesystem::path executableDirPath = NovelRT::Utilities::Misc::getExecutableDirPath();
```

Comment:
This just uses the C++ 17 APIs to combine paths. Ideally, we would have a centralized asset manager that helps manage and resolve these paths (e.g. https://github.com/novelrt/NovelRT/pull/81).